framework/web/GroupUrlRule.php

```diff
@@ -90,9 +90,16 @@ protected function createRules()
         $rules = [];
         foreach ($this->rules as $key => $rule) {
             if (!is_array($rule)) {
+                $verbs = 'GET|HEAD|POST|PUT|PATCH|DELETE|OPTIONS';
+                $verb = null;
+                if (preg_match("/^((?:($verbs),)*($verbs))\\s+(.*)$/", $key, $matches)) {
```

Comment:
There's no need to capture what's not used:

```
^((?:(?:$verbs),)*(?:$verbs))\s+(.*)$
```

Then verb will be in $matches at index 1 and the rest at 2.

Reply:
Changed, thank you